embedchain/vectordb/chroma.py

```diff
@@ -231,7 +231,12 @@ def query(self, input_query: List[str], n_results: int, where: Dict[str, any], s
                 " embeddings, is used to retrieve an embedding from the database."
             ) from None
         results_formatted = self._format_result(result)
-        contents = [result[0].page_content for result in results_formatted]
+        contents = []
+        for result in results_formatted:
+            content_metadata = result[0].metadata
+            content_source = content_metadata.get("url", "Source not found.")
+            content_doc_id = content_metadata.get("doc_id", "Doc id not found.")
```

Comment:
Both `url` and `doc_id` should always be present. I think we should error out whenever this is not the case. Adding some default values is not good and this will fail silently.

Reply:
Updated.